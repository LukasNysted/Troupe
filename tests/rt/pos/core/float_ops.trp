let val _ =  print (1/2 , 3 / 4, 5 div 2, 5 mod 2)
    val x = 32/15 
    val y = 32/17
    val ops_x = (ceil x, floor x, round x)
    val ops_y = (ceil y, floor y, round y)
in print ops_x; print ops_y
end
