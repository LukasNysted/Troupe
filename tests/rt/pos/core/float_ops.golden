2020-03-03T22:59:26.210Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
(0.5, 0.75, 2, 1)
(3, 2, 2)
(2, 1, 2)
>>> Main thread finished with value: ()@{}%{}
